sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1021,7 +1023,8 @@ static int create_sdw_dailink(struct snd_soc_card *card,
 			      struct snd_soc_codec_conf *codec_conf,
 			      int codec_count, int *link_id,
 			      int *codec_conf_index,
-			      bool *ignore_pch_dmic)
+			      bool *ignore_pch_dmic,
+			      bool complex_link_name_required)
```

Comment:
"complex" is scary for reviewers :-)

You may want something that describes just the functionality such as 'append_codec_name' or something.

Reply:
> "complex" is scary for reviewers :-)
> 
> You may want something that describes just the functionality such as 'append_codec_name' or something.

ack